docs-md/operate-and-deploy/installation/check-ksqldb-server-health.md

```diff
@@ -80,4 +80,29 @@ If you're running ksqlDB server in a Docker container, run the
 the `ksql-server` container is `Up`. Check the health of the process in
 the container by running `docker logs <ksql-server-container-id>`.
 
-Page last revised on: {{ git_revision_date }}
+Check a headless ksqlDB Server running in a Docker Container 
+------------------------------------------------------------
+
+A headless deployment of ksqlDB Server doesn't have a REST server that you can
+query for health. Instead, you can probe the JMX port for liveness. A JMX probe
+is the most reliable way to determine readiness of a headless deployment.
+
+The following command probes the JMX port by using the Netcat utility.
+
+```bash
+nc -z <ksql-node>:1099
```

Comment:
Would using the built in script `ksql-print-metrics <port=NNNN>` be an alternative? Not sure it's any better, but it's an out-of-the-box option to consider.

Reply:
@mikebin I made this more prominent earlier in the topic.